src/main/webapp/app/exercises/shared/dashboards/tutor/tutor-exercise-dashboard.component.html

```diff
@@ -281,6 +286,7 @@ <h4 *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus ===
                         </tr>
                         <tr *ngIf="unassessedSubmission && unassessedSubmission?.id > 0 && !exercise.teamMode">
                             <td></td>
+                            <td *ngIf="!!exercise.exerciseGroup"></td>
```

Comment:
should it here not also be `{{unassessedSubmission.id}}`?

Reply:
I don't think so, this is intended